src/main/java/com/ibm/devops/connect/CloudPublisher.java

```diff
@@ -295,11 +295,12 @@ public static boolean uploadQualityData(HttpEntity entity, String url) throws Ex
 
         try {
             HttpPost postMethod = new HttpPost(url);
+            postMethod.setHeader("Authorization", "UserAccessKey " + Jenkins.getInstance().getDescriptorByType(DevOpsGlobalConfiguration.class).getApiToken());
             postMethod.setEntity(entity);
 
             response = httpClient.execute(postMethod);
             resStr = EntityUtils.toString(response.getEntity());
-            if (response.getStatusLine().toString().contains("201")) {
+            if (response.getStatusLine().toString().contains("200")) {
```

Comment:
why is this changing? did it change on the api side?

Reply:
We now hit the REAPI instead of hte RC. The REAPI gives a 200 where as the RC gave a 201